docs/contents/userguide.rst

```diff
@@ -101,25 +101,41 @@ I/O
 ===
 The input and output (I/O) plugin is concerned with loading and saving data
 tables.
+Supported input formats are ``*.csv`` and ``*.pkl/*.pkl.gz``.
+
+AgeTrends
+=========
+This plugin will show MUSE and WMLS volumes as a function of age.
+The data points are colored by category in categorical variables found in the
+data set.
+In the top left drop-down menu, the region of interest can be selected.
+The text field of the menu can be used to search for variable names.
+If a suitable ``neuroHarmonize`` model is loaded (plugin ``Harmonization``),
+the harmonized muse volumes (i.e. ``H_MUSE_Volume_*``) will also show the
+age-conditional normative range for an average intracranial volume.
+
+.. figure:: images/ScreenshotAgeTrend.png
+
 
 MUSE Harmonization
 ==================
 TODO: Add screenshot and description
```

Comment:
@AbdulkadirA I guess you missed this one.

Reply:
@melhemr Could you merge this PR before you add yours?